tests/rptest/tests/datalake/simple_connect_test.py

```diff
@@ -0,0 +1,180 @@
+                            "seed_brokers": self.redpanda.brokers_list(),
+                            "topic": topic,
+                        },
+                        "processors": [{
+                            "schema_registry_encode": {
+                                "url":
+                                self.redpanda.schema_reg().split(",")[0],
+                                "subject": subject,
+                                "refresh_period": "10s"
+                            }
+                        }]
+                    }, {
+                        "file": {
+                            "path": out_path,
+                            "codec": "lines"
+                        }
+                    }]
+                }
+            }
+        }
+
+    def setUp(self):
+        pass
+
+    def _create_schema(self, subject: str, schema: str, schema_type="avro"):
+        rpk = RpkTool(self.redpanda)
+        with tempfile.NamedTemporaryFile(suffix=f".{schema_type}") as tf:
+            tf.write(bytes(schema, 'UTF-8'))
+            tf.seek(0)
+            rpk.create_schema(subject, tf.name)
+
+    @cluster(num_nodes=6)
+    @matrix(cloud_storage_type=supported_storage_types())
+    def test_translating_avro_serialized_records(self, cloud_storage_type):
+        topic_name = "ducky-topic"
+        with DatalakeServices(self.test_context,
+                              redpanda=self.redpanda,
+                              filesystem_catalog_mode=False,
+                              include_query_engines=[QueryEngineType.TRINO
+                                                     ]) as dl:
+
+            dl.create_iceberg_enabled_topic(
+                topic_name,
+                partitions=1,
+                replicas=3,
+                iceberg_mode="value_schema_id_prefix")
+
+            self._create_schema("verifier_schema", self.verifier_schema_avro)
+            connect = RedpandaConnectService(self.test_context, self.redpanda)
+            connect.start()
+
+            output_file = os.path.join(RedpandaConnectService.PERSISTENT_ROOT,
+                                       "verifier.log")
+            # create a stream
+            connect.start_stream(name="ducky_stream",
+                                 config=self.avro_stream_config(
+                                     topic_name,
+                                     "verifier_schema",
+                                     out_path=output_file))
+
+            connect.wait_for_stream_to_finish("ducky_stream")
+
+            cl = dl.catalog_service.client()
+            for n in cl.list_namespaces():
+                self.logger.info(f">>> namespace: {n}")
+            for t in cl.list_tables("redpanda"):
+                self.logger.info(f">>> table: {t}")
+
+            dl.wait_for_translation(topic_name, msg_count=1000)
+            connect.nodes[0].account.copy_from(output_file,
+                                               "/tmp/verifier.log")
```

Comment:
nit: may be worth copying this to the test results dir (like chaos tests do) instead of a temp file for the ease of debugging